src/backend/connection.go

```diff
@@ -84,6 +86,7 @@ func NewConnection(log *xlog.Log, pool *Pool) Connection {
 func (c *connection) Dial() error {
 	var err error
 	defer mysqlStats.Record("conn.dial", time.Now())
+	defer monitor.BackendConnectionInc(c.address)
```

Comment:
It is one bug. if the conn is bad and closed, but it be added. please review again.

Reply:
if the conn is bad and closed,  then call close() ,the added variable will be decremented